src/types/orderOption.graphql

```diff
@@ -1,57 +1,142 @@
 type OrderOption {
   "The list of concepts and any data on the relationship between this order option and other permitted concepts"
   associationDetails: JSON
+
   "The concept id created by CMR for this orderOption"
   conceptId: String!
+
   "The description for this order option"
   description: String
+
   "The ECHO Forms schema for this order option"
   form: String
-  "The guid-id for this order option"
-  id: String!
+
   "The orderOption name."
   name: String
-  "The native id to set on the order option."  
+
+  "The native id to set on the order option."
   nativeId: String
-  "The two levels of order options, provider or system"
+
```

Comment:
Can we also add revision-date here. This date is needed in the orderOptions list. 

```
  "A date when the order option was updated"
  revisionDate: String
```